src/components/SpaceSidebarFooter.vue

```diff
@@ -0,0 +1,62 @@
+<script setup>
+defineProps({
+  space: Object
+});
+</script>
+
+<script>
+export default {
```

Comment:
Thanks for you PR. This should all be written in `script setup` with composition API syntax, like we have in all other components.

Reply:
removed old vue syntax